src/vcpkg/export.cpp

```diff
@@ -486,12 +482,7 @@ namespace vcpkg::Export
     {
         const auto cmake_toolchain = prefix / "scripts/buildsystems/vcpkg.cmake";
         const CMakeVariable cmake_variable = CMakeVariable("CMAKE_TOOLCHAIN_FILE", cmake_toolchain.generic_u8string());
-        print2("\n"
-               "To use the exported libraries in CMake projects use:"
-               "\n"
-               "    ",
-               cmake_variable.s,
-               "\n\n");
+        msg::println(msg::format(msgUsingExportedLibs).append_indent().append_raw(cmake_variable.s).append_raw("\n\n"));
```

Comment:
```suggestion
        msg::println(msg::format(msgUsingExportedLibs).append_indent().append_raw(cmake_variable.s).append_raw("\n"));
```

`printnl` already adds a `\n`